Testreglar/2.4.4/Nett/nett-2.4.4a.json

```diff
@@ -0,0 +1,232 @@
+{
+	"namn": "Nett-2.4.4a Formål med lenker går fram av lenketeksten, eller lenketeksten og konteksten saman 2023",
+	"id": "nett-2.4.4a",
+	"testlabId": 473,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<p>Lenker som inneholder tekst oppfyller kravet dersom:</p>\r\n<ul>\r\n<li>Lenketeksten alene beskriver formålet med lenken eller</li>\r\n<li>Lenketeksten sammen med programmeringsmessig bestemt lenkekontekst beskriver formålet med lenken. For programmatisk bestemt lenkekontekst vurderes tekst som står i\r\n<ul>\r\n<li>samme avsnitt, liste eller celle i en tabell som lenken gjør eller tilsvarende eller</li>\r\n<li>en tabelloverskrift som er knyttet til tabellcellen som lenken står i eller tilsvarende</li>\r\n</ul>\r\n</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.7"
+		},
+		{
+			"title": "3.8"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva side testar du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsida lenker som inneheld tekst?",
+			"ht": "<p>Gjer ein visuell inspeksjon, og sjekk om det finst lenker med tekst på testsida.</p>\n<p><strong>Merk: </strong></p>\n<ul>\n<li>Lenkas plassering er utan betydning, den kan for eksempel ligge i ein meny, liste eller tabell.</li>\n<li>Du skal ikkje teste knappar eller lenker som berre inneheld bilde. Dette blir testa i SK 4.1.2 og SK 1.1.1.</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testside har ikkje lenker som inneheld tekst."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Kva lenke testar du?",
+			"ht": "<p>Beskriv lenka, slik at det er mogleg å identifisere den i ettertid. Om du finn fleire lenker registrerer du ei og ei.</p>\n<p><strong>Tips:</strong> Du kan skrive lenketekst, plassering på side, element som er i nærleiken eller liknande. </p>",
```

Comment:
Her bør vi ha en "standard" beskrivelse slik at vi gjør det likt. Hvordan dette standardiseres er ikke så viktig for meg, så lenge det blir standardisert.